File: 118. Pascal's Triangle/main.py
class Solution:
    def generate(self, numRows: int) -> List[List[int]]:
        res = [[1] for i in range(numRows)]

        for i in range(1, numRows):
            for j in range(i-1):
                res[i].append(res[i-1][j]+res[i-1][j+1])

            res[i].append(1)
        
        return res

File: 118. Pascal's Triangle/main.js
/**
 * @param {number} numRows
 * @return {number[][]}
 */
var generate = function (numRows) {
  let res = [[1]];

  for (let i = 1; i < numRows; i++) {
    res.push([1]);
    for (let j = 0; j < i - 1; j++) {
      res[i].push(res[i - 1][j] + res[i - 1][j + 1]);
    }
    res[i].push(1);
  }

  return res;
};
